modules/upgrade/proc-upgrade-rhdh-helm.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-upgrade-rhdh-helm_{context}"]
+
+= Upgrading the {product} Helm Chart
+
+You can upgrade to a later version of {product} in {ocp-short} by using either the web console or the CLI.
+
+* {ocp-short} web console
+
+. In the *Developer* perspective, click *Helm* to open the *Helm Releases* tab.
+
+. Click the overflow menu on the Helm release that you want to use and select *Upgrade*.
+
+. On the *Upgrade Helm Release* page, select the version of {product-short} that you want to upgrade to from the chart version drop-down list.
+
+. Click *Upgrade*.
++
+[NOTE]
+====
+It might take a few minutes to delete the resources in the older versions and to start the newer versions of the {product-short} pods.
+====
+
+. Close all open {product-short} web pages, and log in again to verify that the upgrade was successful.
+
+* {ocp-short} CLI
+
+. Log in to the {ocp-short} cluster as the cluster administrator and switch to the project or namespace in which {product-short} was installed.
++
+[source]
+----
+oc login -u <user> -p <password> https://api.<HOSTNAME>:6443
+oc project <your_namespace>
+----
+
+. For a new version of the {product-short} Helm chart, run the following upgrade command:
++
+[source,subs="attributes+"]
```

Comment:
Should this be [source, yaml, subs="attributes+"]

Reply:
The text in this file was not changed nor created under this PR, as the scope of this PR is only a relocation of this existing content. IMO, this code block is not even necessary; we might consider removing it in the future.

Reply:
Yes, it should be